videos.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Videos</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <nav class="container">
+      <div class="nav-left container">
+        <!--icons left-->
+        <i class="fa-solid fa-bars-staggered"></i>
+        <!--styled together-->
+        <img
+          src="./images/play.png"
+          alt="play"
+          class="play"
+          width="80"
+          height="80"
+        />
+        <span class="logo-name">MyTube</span>
+      </div>
+      <!--middle icons and search bar-->
+      <div class="nav-middle container">
+        <div class="search-box container">
+          <input type="text" placeholder="Search" />
+          <i class="fa-solid fa-magnifying-glass"></i>
+        </div>
+        <i class="fa-solid fa-microphone"></i>
+      </div>
+      <!--icons right-->
+      <div class="nav-right container">
+        <i class="fa-solid fa-square-plus"></i>
+        <i class="fa-solid fa-bell"></i>
+        <img src="./images/me.jpg" alt="me" class="user-icon" />
+      </div>
+    </nav>
+
```

Comment:
For higher SEO and semantic reasons, i think you should wrap your navigation inside header tags

Reply:
- [ ] I think it would be better to avoid stating similar comments for the same lines of code in a single project, :alien:  you can achieve this with the previous comment to make your review more professional :joy: :joy: